app/models/hardware.rb

```diff
@@ -154,4 +154,11 @@ def m_snapshots(parent, xmlNode, _deletes)
   def m_volumes(parent, xmlNode, _deletes)
     Volume.add_elements(parent, xmlNode)
   end
+
+  def sockets
+    return 0 unless logical_cpus && cores_per_socket
+    logical_cpus / cores_per_socket
```

Comment:
@bdunne Can you please take a look at this calculation for determining the number of physical sockets on a host and let us know if you think it's correct?